build/after-pack.js

```diff
@@ -32,7 +32,7 @@ exports.default = async function (context) {
         'Filecoin Station.app',
         'Contents',
         'Resources',
-        'core'
+        'checker'
```

Comment:
Since Filecoin Station will be eventually called Checker App or simply Checker, I think it would be better to call this directory `checker-node`. 

Such a name is also more consistent with the filename `main/checker-node.js` I see later in this pull request.

WDYT?

Reply:
If you agree with my proposal, then please remember to update other files too, e.g. `.eslintrc.json`, `.gitignore` and `electron-builder.yml`.

Reply:
> Since Filecoin Station will be eventually called Checker App or simply Checker

For correctness: "Filecoin Station" will not be renamed to Checker, it is becoming "Checker Network". "Filecoin Station Desktop" is becoming "Checker App" or short "Checker".

Reply:
a301a49